core/lib/spree/testing_support/factories/promotion_factory.rb

```diff
@@ -48,11 +52,8 @@
         adjustment_rate { 10 }
       end
 
-      after(:create) do |promotion, evaluator|
-        calculator = Spree::Calculator::FlatRate.new
-        calculator.preferred_amount = evaluator.adjustment_rate
-        Spree::Promotion::Actions::CreateItemAdjustments.create!(calculator: calculator, promotion: promotion)
-      end
+      with_adjustable_action
+      preferred_amount { adjustment_rate }
```

Comment:
It would help get the whole context at each read for sure, but if we have a default, and it's better to be explicit even when the value is the same as the default, I'd start questioning what the purpose of having a default is? 🤔 

So my personal preference is let's skip them, but I don't have strong opinions and I think both ways have pros and cons. Up to you!

Reply:
Let's skip them 👍